docs/backup_restore/catalog-db-restore.sh

```diff
@@ -85,8 +76,6 @@ cf unbind-service catalog-fetch catalog-db-venerable
 cf bind-service catalog-fetch catalog-db
 cf restart catalog-fetch
 
-# reindex solr
-cf run-task catalog-admin -c "ckan search-index rebuild -i -o" --name search-index-rebuild -k 2G -m 2G
-
 # cleanup
-cf delete-service catalog-db-venerable
\ No newline at end of file
+cf delete-service catalog-db-venerable
```

Comment:
we can include `-f` to delete without confirmation if we want here. otherwise we'll need to periodically checkin and confirm when prompted